Week2/QA_PREP_EXERCISE.md

```diff
@@ -0,0 +1,32 @@
+As a preparation step for the upcoming Q&A, you need to work on the following exercise, which is based on the prep exercise of the previous week. 
+
+### Exercise
+
+Last week you designed a database for storing food recipes. How can you improve your tables, based on what you learned this week?
+
+- What changes do you need to make to include [these new recipes](recipes.md) in your database?
+
+- Where can you add a primary key?
+- Which tables should be linked with a foreign key?
+- What type of relationships do you see between the tables?
+
+- Can you make queries to get:
+    > All the vegetarian recipes with potatoes
+    > All the cakes that do not need baking
+    > All the vegan and Japanese recipes
+
+Since your table data might be different from the above questions, feel free to improvise.
+Make sure that your database is running locally and that your queries are reproducable in .js files. 
```

Comment:
@robvk would this be sufficient as a requirement?

Reply:
Yes, think that is clear!